examples/classification/main.py

```diff
@@ -157,8 +158,8 @@ def autoq_eval_fn(model, eval_loader):
 
     model.to(config.device)
 
-    resuming_model_sd, resuming_checkpoint = load_resuming_checkpoint(resuming_checkpoint_path)
-    compression_ctrl, model = create_compressed_model(model, nncf_config, resuming_state_dict=resuming_model_sd)
+    nncf_checkpoint, resuming_checkpoint = load_resuming_checkpoint(resuming_checkpoint_path)
+    compression_ctrl, model = create_compressed_model(model, nncf_config, nncf_checkpoint=nncf_checkpoint)
```

Comment:
Intentionally rename `resuming_state_dict` to `nncf_checkpoint` in the `create_compressed_model` method to avoid loading in a deprecated manner:
```python
model_state_dict = compressed_model.state_dict()
create_compressed_model(model, config, resuming_state_dict=model_state_dict)
```
It should be deprecated since model's state_dict, i.e. torch tensors, is not enough to unambiguously resume compressed model. We need loading of builder and controller states, which are part of NNCF checkpoint: